package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Note that in this file I will have to update the `@armrest-client` versioning once the corresponding [`armrest-node` PR](https://github.com/payload-code/armrest-node/pull/6) is merged